frame/cqlvalue.go

```diff
@@ -198,6 +199,31 @@ func (c CqlValue) AsFloat64() (float64, error) {
 		uint64(c.Value[7])), nil
 }
 
+func (c CqlValue) AsTextSet() ([]string, error) {
+	if c.Type.ID != SetID || c.Type.List.Element.ID != VarcharID {
```

Comment:
I get errors when using this function. There should be:
`c.Type.Set.Element.ID`
instead of:
`c.Type.List.Element.ID`